pkg/revocation/revocation_db.go

```diff
@@ -49,38 +71,38 @@ func NewRevocationDB(revocationDBURL string) (*RevocationDB, error) {
 	return db, nil
 }
 
-// newRevocationDBBolt creates a bolt-backed RevocationDB
-func newRevocationDBBolt(path string) (*RevocationDB, error) {
+// newRevocationDBBolt creates a bolt-backed DB
+func newRevocationDBBolt(path string) (*DB, error) {
```

Comment:
I noticed `NewRevocationDB` was changed to `NewDB` so why not make these `newBoltDB` and `newRedisDB`? very nit though

Reply:
Good call. Fixed.